modular_skyrat/master_files/code/modules/client/preferences.dm

```diff
@@ -50,6 +50,14 @@
 	//Determines if the player has undergone TGUI preferences migration, if so, this will prevent constant loading.
 	var/tgui_prefs_migration = TRUE
 
+	///The vore prefs datum
+	var/datum/vore_prefs/vr_prefs
+
+/datum/preferences/New(client/C)
+	. = ..()
+	if (!vr_prefs)
+		vr_prefs = new(C, src)
```

Comment:
you need to clean this up on Destroy, otherwise you're leaving refs floating and causing hard dels